src/client/pages/Thema/HLI/HLI-thema-config.ts

```diff
@@ -5,10 +5,142 @@ import { generatePath } from 'react-router';
 import { HLIRegelingFrontend } from '../../../../server/services/hli/hli-regelingen-types';
 import { IS_PRODUCTION } from '../../../../universal/config/env';
 import { dateSort } from '../../../../universal/helpers/date';
-import { LinkProps } from '../../../../universal/types/App.types';
 import { DisplayProps } from '../../../components/Table/TableV2.types';
 import { MAX_TABLE_ROWS_ON_THEMA_PAGINA } from '../../../config/app';
-import type { ThemaRoutesConfig } from '../../../config/thema-types';
+import { propagateFeatureToggles } from '../../../config/buildFeatureToggle';
+import type {
+  PageConfig,
+  ThemaConfigBase,
+  WithDetailPage,
+} from '../../../config/thema-types';
+
+const THEMA_TITLE = 'Stadspas en regelingen bij laag inkomen' as const;
+
+type HLIThemaConfig = ThemaConfigBase &
+  WithDetailPage &
+  PageConfig<'detailPageStadspas'> &
+  PageConfig<'specificatieListPage'> &
+  PageConfig<'regelingenListPage'>;
+
+export const themaConfig = {
+  id: 'HLI' as const,
+  //TODO MIJN-12205
+  title: THEMA_TITLE,
+
+  featureToggle: propagateFeatureToggles({
+    active: true,
+    stadspas: {
+      // // BUG: ? When this is false, i expect: 1. Not to see stadspas products on the themepage 2. Not to see "Stadspas" in the title
+      active: true,
+      blokkerenActive: true,
+      deblokkerenActive: !IS_PRODUCTION,
+    },
+    regelingen: {
+      // BUG: ? When this is false, i expect: 1. Not to see the RTM document on the theme page. 2. Not to see "regelingen bij laag inkomen" in the title
+      active: true,
+      enabledCZM: false,
```

Comment:
Pas op deze staat nu uit!
```suggestion
      enabledCZM: true,
```
Je kan beter direct op de testen brancht dit soort dingen committen en pushen. Dan voorkom je dat het perongeluk hier meekomt.